### PR TITLE
release: changelog and version tag for 1.0.0-bc14

### PR DESCRIPTION
**Pull request by karencfv**
## Description
Updates version tag, corresponding changelog and release documentation.

### CODE REVIEW

File: notes/1.0.0-bc14.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Changelog
+
+## Breaking changes
```
Comment (marclop) [inline notes/1.0.0-bc14.md]: We can remove this,  there's no breaking changes.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 export GO111MODULE ?= on
-export VERSION ?= 1.0.0-bc10
+export VERSION ?= 1.0.0-bc14
```
Comment (marclop) [inline Makefile]: Looking at the tags and how `go mod` operates, it should have the `v` prefix.
```suggestion
export VERSION ?= v1.0.0-bc14
```

File: notes/1.0.0-bc14.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#### api: Add CloudClientRuntime client.Runtime wrapper [#46](https://github.com/elastic/cloud-sdk-go/pull/46)
+
+Adds a new structure which implements runtime/client.ClientTransport
+and allows predefined rules to call a regionless or region prefixed URL
+API client.
+
+This change is necessary since the /deployments path does not work
+with the region interpolated in the URL.
+
+### Docs
```
Comment (marclop) [inline notes/1.0.0-bc14.md]: Same here

File: notes/1.0.0-bc14.md (hunk 1 above)
Comment (karencfv) [reply notes/1.0.0-bc14.md]: :P

File: notes/1.0.0-bc14.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Changelog
```
Comment (marclop) [inline notes/1.0.0-bc14.md]: sorry, this file needs to be renamed

**Pull request merged**